Framework/PythonInterface/mantid/api/src/Exports/SpectrumDefinition.cpp

```diff
@@ -0,0 +1,36 @@
+#include "MantidTypes/SpectrumDefinition.h"
+
+#include <boost/python/class.hpp>
+#include <boost/python/tuple.hpp>
+
+using Mantid::SpectrumDefinition;
+using namespace boost::python;
+
+// Helper function to convert a std::Pair to a Python tuple
+boost::python::tuple convertPairToTuple(const SpectrumDefinition &self,
```

Comment:
I don't think this name conveys what you are trying to do here. This should be called `getSpectrumPair` or something better. Maybe @OwenArnold has a better idea.

Reply:
Also, if you look in the documentation for the Instrument Access Layers, you may want to add some python examples of how `SpectrumInfo` can be used.

Reply:
I would certainly advocate dropping the the `pair` part. Maybe `toTuple` is enough? I notice that this is bound to a python `get` method name.

Reply:
Sounds good to me - I'll change the name

Reply:
Should I change the `get` method name to something like `getDetectorIndexTimeIndexTuple` or should I leave it as is?